### PR TITLE
Performance measurement

### PR DESCRIPTION
**Pull request by vivek-dhayalan**

### CODE REVIEW

File: learn/react-native/flashlight.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Flashlight"
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/react-native/flashlight.md:2]: Change title for better SEO. Write what flashlight is used for

File: learn/react-native/performance-measurment.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Performance Measurement"
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/react-native/performance-measurment.md:2]: Performance Measurement of React Native Mobile apps

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]
After fixing the comments i noted, it can be merged. Looks good to me.

File: learn/react-native/performance-measurment.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+It enables developers to identify and rectify issues promptly, safeguarding user satisfaction and retention.
+Moreover, performance data guides informed decision-making, helping to optimize resource allocation and enhance overall
+app functionality. Ultimately, diligent performance measurement supports revenue growth, protects brand reputation, and
+provides a competitive edge in the bustling digital marketplace.
+
+So, let's see the various performance metrics and tools used to measure them in detail with below sections:
+
+- [Performance Metrics](https://docs.wavemaker.com/learn/react-native/performance-metrics)
+- **Performance Tools** 
+  - [Flashlight](https://docs.wavemaker.com/learn/react-native/flashlight)
```
Comment (swethakundaram) [inline learn/react-native/performance-measurment.md:20]: this link is broken

File: website/sidebars.json [hunk 4]
```diff
@@ -669,7 +669,8 @@
 						"react-native/generate-web-preview-locally",
 						"react-native/web-preview-debug",
 						"react-native/expo-debug",
-						"react-native/flipper"
+						"react-native/flipper",
+						"react-native/performance-measurement"
```
Comment (swethakundaram) [inline website/sidebars.json:673]: other new docs are missing on the sidenav bar

Comment (vivek-dhayalan) [reply website/sidebars.json:673]: The idea is to have the Performance Measurement as main page and other pages can be accessed through navigation.

File: learn/react-native/flashlight.md (hunk 1 above)
Comment (vivek-dhayalan) [reply learn/react-native/flashlight.md]: Updated

File: learn/react-native/performance-measurment.md (hunk 2 above)
Comment (vivek-dhayalan) [reply learn/react-native/performance-measurment.md]: updated

File: website/sidebars.json (hunk 4 above)
Comment (vivek-dhayalan) [reply website/sidebars.json]: updated

**Pull request merged**